config/locales/de.events.yml

```diff
@@ -56,6 +56,13 @@ de:
       unnecessary: "nicht benötigt"
       available: "vorhanden"
       unavailable: "nicht vorhanden"
+      print_habits: "Essgewohnheiten drucken (PDF)"
+      print_title: "Essgewohnheiten der Teilnehmer für %{title}"
+      print_summary: "Insgesamt werden wir bei dieser Veranstaltung %{number} Teilnehmer/-innen haben."
+      print_summary_vegetarian: "%{number} Personen ernähren sich vegetarisch."
+      print_summary_vegan: "%{number} Personen ernähren sich vegan."
+      print_summary_omnivorous: "%{number} Personen ernähren sich omnivor."
```

Comment:
Doesn't sound right if `number` is 1. I18n has a feature for adding separate singular and plural translations, you could use that.